Recipely/Recipely/FavoritesScreen/View/FavoritesViewController.swift

```diff
@@ -0,0 +1,15 @@
+// FavoritesViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран с фаворитами
```

Comment:
```suggestion
/// Экран с фаворитами
```
чьими фаворитами может избранного